src/components/Sidebar/Sidebar.css

```diff
@@ -0,0 +1,20 @@
+.sidebar {
+    background-color: var(--bg-color);
+    position: sticky;
+    min-width: 13rem;
+    height: 88vh;
```

Comment:
`position: sticky` won't be effective until a atleast of the top bottom left right is applied

Reply:
I'll make changes